PyBitTorrent/bcoder.py

```
@@ -47,8 +49,13 @@ def get_dict(data) -> Dict:
     key: bytes = bdecode(data)
     dictionary = {}
     while key:
-        value = bdecode(data)
-        dictionary[key] = value
+        if type(key) == dict :
```

Whats the purpose of this check?

---

`bdecode()` can also return dictionaries. In this case it wasn't supposed to happen, but some torrent files that I tested don't follow the standard, making `bdecode()` return a dictionary in some cases. Maybe raise an exception for corrupted files would be more elegant?

---

I rather to return a dict and handle this case instead of throwing exception, great work